### PR TITLE
Update unit test configuration

### PR DESCRIPTION
**Pull request by gitromba**
Most of the global constants are not having valid value which breaks some dependent tests. 

This PR updates some values with valid values in order to fix some related issues. 

Another PR will shortly follow in `tao-core`.
EDIT: related PR: https://github.com/oat-sa/tao-core/pull/2675 and this one should be merged upfront.

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '13.4.0',
+    'version' => '13.4.2',
```
Comment (gabrielfs7) [inline manifest.php:35]: Why 2 bugfix versions and not `13.4.1`?

File: test/bootstrap.php [hunk 2]
```diff
@@ -26,4 +26,4 @@
 // Use TAO dependency resolver first.
 require_once $extensionRoot . './../vendor/autoload.php';
 
-common_Config::load($extensionRoot . 'config/sample/generis.conf.php');
+common_Config::load($extensionRoot . 'test/generis.conf.php');
```
Comment (gabrielfs7) [inline test/bootstrap.php:29]:
```suggestion
common_Config::load($extensionRoot . 'test/config/generis.conf.php');
```
Wdyt?

File: manifest.php (hunk 1 above)
Comment (gitromba) [reply manifest.php:35]: I've opened this one from another branch: https://github.com/oat-sa/generis/pull/836.

It contains some constant value updates which are not BC that's it should be done in 2 (actually 3 as there's another one in `tao-core`) steps.

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful ⚠️ consider mentioning what exactly was updated
- [x] Pull request's target is not `master`

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
LGTM. Good job! :)

File: test/config/generis.conf.php [hunk 3]
```diff
@@ -36,11 +36,12 @@
...
 # platform identification
 define('GENERIS_INSTANCE_NAME', '');
-define('GENERIS_SESSION_NAME', '');
+define('GENERIS_SESSION_NAME', 'test');
 
 # paths
 define('ROOT_PATH', '');
-define('ROOT_URL', '');
+// trailing slash is important!
+define('ROOT_URL', 'https://test.com/');
```
Comment (mike-ionut-mihai-sandu-tao) [inline test/config/generis.conf.php:44]: maybe we can use demo.taotesting.com ? 
we will always have that subdomain and it's free publicity

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: test/config/generis.conf.php (hunk 3 above)
Comment (gitromba) [reply test/config/generis.conf.php:44]: As long as it's a valid domain it doesn't really matter the value. I like it though having a "real" value. Thanks @mike-ionut-mihai-sandu-tao!

**Pull request merged**